README.md

```diff
@@ -1,21 +1,23 @@
 # GitOps Engine
+<div align="center">
 
-![logo](docs/img/Argo-Flux_logos.png)
+![image](https://user-images.githubusercontent.com/426437/82109570-f6c7ed80-96eb-11ea-849c-2bd5fe89b571.png)
 
-Two of the biggest GitOps projects are joining forces. [Argo CD](https://github.com/argoproj/argo-cd) and [Flux CD](https://github.com/fluxcd/flux) have been out there for years, making the lives of operators and organisations easier. Now the teams are building the best possible GitOps solution together. The first step on this journey is the GitOps Engine.
+</div>
 
-Argo and Flux already have similar sets of functionality, soon they will start sharing code using the GitOps Engine. The GitOps Engine will be responsible for the following functionality:
+Various GitOps operators address different use-cases, provide different user experience but all have the same set of features in the core. Teams behind
+[Argo CD](https://github.com/argoproj/argo-cd) and [Flux CD](https://github.com/fluxcd/flux) joining forces to implement the reusable library that implements core GitOps
+features and make it available for everyone. This repository is intended to host the implementation of for the following core GitOps features:
 
+- Kubernetes resource cache ✅
+- Resources reconciliation ✅
+- Sync Planning ✅
 - Access to Git repositories
-- Kubernetes resource cache
 - Manifest Generation
-- Resources reconciliation
-- Sync Planning
-
-Over time, this might be useful to other projects too.
 
 ## Proposals, specifications and ideas
 
+Would you want to propose one more feature and want to enhance the existing one?
```

Comment:
Do you want to propose one more feature and want to enhance the existing one?

Reply:
done